src/etc/htmldocck.py

```diff
@@ -632,14 +596,10 @@ def check_command(c, cache):
             else:
                 raise InvalidCheck("Invalid number of {} arguments".format(c.cmd))
 
-        elif c.cmd == "valid-html":
-            raise InvalidCheck("Unimplemented valid-html")
-
-        elif c.cmd == "valid-links":
-            raise InvalidCheck("Unimplemented valid-links")
-
         else:
-            raise InvalidCheck("Unrecognized {}".format(c.cmd))
+            # Ignore unknown directives, they might be compiletest directives.
+            # In any case, compiletest rejects unknown directives for us.
+            return
```

Comment:
Suggestion: maybe additionally add that

```suggestion
            # Ignore unknown directives, they might be compiletest directives.
            # In any case, compiletest rejects unknown directives for us.
            #
            # This relies on the convention that both compiletest and htmldocck
            # directives share the same `//@` prefix.
            return
```